File: Entity/Quartier.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BrasilBurger.Web.Entity
{
    [Table("quartier")]
    public class Quartier
    {
        [Column("id")]
        public int Id { get; set; }
        
        [Column("nom")]
        public string Nom { get; set; }
        
        [Column("id_zone")]
        public int? IdZone { get; set; }
        
        [ForeignKey("IdZone")]
        public Zone? Zone { get; set; }
    }
}

File: src/Entity/Quartier.php
<?php
namespace App\Entity;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'quartier')]
class Quartier
{
    #[ORM\Id, ORM\GeneratedValue, ORM\Column]
    private ?int $id = null;
    #[ORM\Column(length: 255, nullable: true)]
    private ?string $nom = null;
    #[ORM\Column(name: 'id_zone', nullable: true)]
    private ?int $idZone = null;

    public function getId(): ?int { return $this->id; }
    public function getNom(): ?string { return $this->nom; }
    public function getIdZone(): ?int { return $this->idZone; }
}
